src/locales/en.json

```diff
@@ -0,0 +1,35 @@
+{
+  "title": {
+    "intro": {
+      "first": "Mobile program for learning ",
+      "second": "the Belarusian",
+      "third": "language"
+    },
+    "func": "Functions",
+    "team": "Team",
+    "connect": "Connect with us"
+  },
+  "subtitle": {
+    "func": {
+      "rules": "Rules and words.",
+      "practice": "Practice.",
+      "prize": "Reward.",
+      "competition": "Leadership."
+    }
+  },
+  "description": {
+    "intro": "Who needs us, if not ourselves?! Definitely not your Dualingo. In October 2022, we gathered a team of volunteers and started developing a mobile application to make it easier to learn our native language and switch to it in everyday life.",
+    "rules": "Lessons are collected by vocabulary topics: Family, Apartment, Shop, etc. Each lesson also has corresponding grammar rules.",
+    "practice": "After each lesson, you can go through its exercises to check how well you learned the lesson.",
+    "prize": "Get bonuses for correctly performed exercises and daily practice without interruptions.",
+    "competition": "Every week you will be able to compete with other users of the program in terms of the number of stars received."
```

Comment:
"Every week you will be able to compete with other users of the app in collecting the most stars."